CMakeLists.txt

```
@@ -72,7 +72,7 @@ ENDFUNCTION()
 LIST(APPEND CMAKE_MODULE_PATH cmake/Modules)
 
 IF(NOT KOKKOS_HAS_TRILINOS)
-  cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+  cmake_minimum_required(VERSION 3.17.1 FATAL_ERROR)
```

```suggestion
  cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
```
unless someone can explain what is wrong with 3.17.0